src/lib.rs

```diff
@@ -137,10 +137,7 @@ pub use util::sighash::SchnorrSigHashType;
 
 pub use util::ecdsa::{self, EcdsaSig, EcdsaSigError};
 pub use util::schnorr::{self, SchnorrSig, SchnorrSigError};
-#[deprecated(since = "0.26.1", note = "Please use `ecdsa::PrivateKey` instead")]
-pub use util::ecdsa::PrivateKey;
-#[deprecated(since = "0.26.1", note = "Please use `ecdsa::PublicKey` instead")]
-pub use util::ecdsa::PublicKey;
```

Comment:
FYI, for the future, this warning does not really work. If you want to deprecate a type, you would have to have to 
- Rename the existing type
- Create a new type `pub type OldName = NewName`
- Add deprecated above it